src/translations/screens/subscreens/Travellers.ts

```diff
@@ -0,0 +1,36 @@
+import {translation as _} from '../../commons';
+import {UserProfileWithCount} from '../../../screens/Ticketing/Purchase/Travellers/use-user-count-state';
+
+const TravellersTexts = {
+  header: {
+    title: _('Velg sone'),
+    leftButton: {
+      text: _('Avbryt'),
+      a11yLabel: _('Avbryt kjøpsprosessen'),
+    },
+  },
+  errorMessageBox: {
+    title: _('Det oppstod en feil'),
+    failedOfferSearch: _('Klarte ikke å søke opp pris'),
+    failedReservation: _('Klarte ikke å reservere billett'),
+  },
+  tariffZones: {
+    a11yHint: _('Aktivér for å velge soner'),
+  },
+  travellerCounter: {
+    text: (u: UserProfileWithCount) => _(u.name.value),
```

Comment:
How would this get translated? Is `u.name` already a translated value? I might not entirely understand how the `LanguageAndText` type is being used, but by the name it would seem like translations might already be defined elsewhere for these properties?

Reply:
This is NeTeX `MultilingualString`. It is an object { lang: string, value: string } where lang is the language code. Generally the `name` field has the value for the default language, and another field `alternativeNames` has the additional languages.

So for `UserProfile` it would look something like this:
```
{ 
  ...
  name: { lang: "nob", value: "Voksen" },
  additionalNames: [
    { lang: "nno", value: "Vaksen" },
    { lang: "eng", value: "Adult" }
  ]
```  

So for english translation we would have to use something like `u.additionalNames.filter(n => n.lang === "eng")?.value`. Probably in a helper method as this has to be done several places.

Reply:
Yes, I see. I suspected something like that. Perhaps this kind of logic also could be placed elsewhere? I'm just asking, though, I think we should agree on a common practice on this. My main argument is keeping the text files as clean as possible for translators